source/install/self-managed-changelog.md

```diff
@@ -4,12 +4,90 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 - [Release v6.2 - Feature Release](#release-v6-2-feature-release)
 - [Release v6.1 - Feature Release](#release-v6-1-feature-release)
-- [Release v6.0 - Feature Release](#release-v6-0-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5.37-extended-support-release)
 
+## Release v6.4 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-02-16**
+
+### Important Upgrade Notes
+ - 
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.3, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Playbook Updates
+
+
+#### Boards Updates
+ - Added Boards support for Global Retention Policy.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated **Account Settings** terminology to **Settings**.
+ - Added Accept-Language header to generate link previews in the default Server language.
+ - The **Invite Members** button is now hidden when the Direct Message category is collapsed.
+ - Added Collapsed Reply Threads (Beta) tour functionality.
+ - Added the current product name information to invoices.
+ - Added support for metered billing invoices.
+ - Added a keyboard shortcut to open and expand the right-hand pane.
+ - UX improvements to the **System console > licensing page**: added a new modal for the upload license workflow.
+
+#### Administration
+ - Improved plugins performance by re-initializing only upon plugin configuration changes.
+ - Removed dead struct ``ManifestExecutables``.
+ - Added support for exporting and importing the post type and ``edit_at`` post details.
+ - Added support for ``postgresql`` schema designator.
+ - New schema migrations.
```

Comment:
> The new migration system will be going to apply all migrations.

It will just check for the existence of all migrations and pass through them right? Not exactly _apply_ all migrations, but _check_ if a migration has happened.

Reply:
cc @isacikgoz

Reply:
Right, just wanted to clarify the language a bit to be extra-clear to the customers :stuck_out_tongue:

Reply:
@isacikgoz - What do you think of:

> A new schema migration system has been introduced. **Before updating the app to this version, it is strongly recommended to have a database backup**. The new migration system will run through all existing migrations to record them to a new table. This will only happen for the first run, to migrate the application to the new system. The table where the migration information is stored is `db_migrations`. Additionally, a `db_lock` table is used to prevent multiple installations from running migrations in parallel. In case of an error while applying the migrations please check this table first. If you encounter an issue please file an issue by including the failing migration name, database driver/version and the error logs

Reply:
Yeah, makes much more sense. @amyblais can we merge @agnivade's suggestion into the important notes?

Reply:
Updated!